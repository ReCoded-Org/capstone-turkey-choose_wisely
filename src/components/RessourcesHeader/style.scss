@import "../../style/variables.scss";
@import "../../style/mixins.scss";

.ressources_page {
  background-color: $light-color;
  height: 50.438rem;
  margin-bottom: 2rem;

  .ressources_title {
    height: 12.188rem;
    width: 31.563rem;
    left: 2.938rem;
    margin-top: 4.563rem;
  }

  .ressouces_page1 {
    height: 30.75rem;
    width: 43.625;
    margin-left: 5.188rem;
    margin-top: 7.063rem;
  }

  .ressouces_page2 {
    height: 26.563rem;
    width: 34.938rem;
    margin-left: 1.25rem;
    margin-top: 4rem;
    margin-bottom: 4.438rem;
  }
}
