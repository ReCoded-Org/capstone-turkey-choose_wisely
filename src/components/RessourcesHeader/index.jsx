import React from "react";
import { Container, Row, Col } from "react-bootstrap";
import SectionTitle from "./../SectionTitle";
import { useTranslation } from "react-i18next";
import "./style.scss";
import RessourcesPage1 from "./../../images/RessourcesPage1.svg";
import RessourcesPage2 from "./../../images/RessourcesPage2.svg";

const RessourcesHeader = () => {
  const { t } = useTranslation();

  return (
    <div className="ressource_page">
      <Container>
        <Row>
          <Col sm={2} md={4}>
            <div className="ressources_title">
              <SectionTitle
                bigTitle={t("ressourcesPage.header.title")}
                subtitle={t("ressourcesPage.header.subtitle")}
              />
              <Row>
                <img
                  alt="Illustration of a man working on a PC"
                  className="ressouces_page2"
                  src={RessourcesPage2}
                  fluid
                />
              </Row>
            </div>
          </Col>
          <Col sm={2} md={4}>
            <img
              alt="Illustration of 2 women planifying taks on a board"
              className="ressouces_page1"
              src={RessourcesPage1}
            />
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default RessourcesHeader;
