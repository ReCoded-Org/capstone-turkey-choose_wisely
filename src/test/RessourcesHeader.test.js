import React from "react";
import RessourcesHeader from "./../components/RessourcesHeader/index.jsx";
import renderer from "react-test-renderer";

test("Snapshop test of RessourcesHeader component", () => {
  const map = renderer.create(<RessourcesHeader />);
  let tree = map.toJSON();

  expect(tree).toMatchSnapshot();
});
