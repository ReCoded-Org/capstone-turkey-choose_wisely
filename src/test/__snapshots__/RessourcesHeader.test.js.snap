// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshop test of RessourcesHeader component 1`] = `
<div
  className="ressource_page"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-4 col-sm-2"
      >
        <div
          className="ressources_title"
        >
          <div
            className="section_title"
          >
            <h1
              className="big_title"
            >
              ressourcesPage.header.title
            </h1>
            <p
              className="subtitle"
            >
              ressourcesPage.header.subtitle
            </p>
            <h2
              className="title"
            />
          </div>
          <div
            className="row"
          >
            <img
              alt="Illustration of a man working on a PC"
              className="ressouces_page2"
              fluid={true}
              src="RessourcesPage2.svg"
            />
          </div>
        </div>
      </div>
      <div
        className="col-md-4 col-sm-2"
      >
        <img
          alt="Illustration of 2 women planifying taks on a board"
          className="ressouces_page1"
          src="RessourcesPage1.svg"
        />
      </div>
    </div>
  </div>
</div>
`;
